state-synchronizer/src/tests/integration_tests.rs

```diff
@@ -168,13 +174,25 @@ impl SynchronizerEnv {
         // The voting power of peer 0 is enough to generate an LI that passes validation.
         for (idx, signer) in signers.iter().enumerate() {
             let voting_power = if idx == 0 { 1000 } else { 1 };
-            let addr: NetworkAddress = "/memory/0".parse().unwrap();
+            let addr = NetworkAddress::from_str("/memory/0").unwrap();
+            let raw_addr = RawNetworkAddress::try_from(&addr).unwrap();
+            let shared_val_netaddr_key = TEST_SHARED_VAL_NETADDR_KEY;
+            let key_version = TEST_SHARED_VAL_NETADDR_KEY_VERSION;
+            let enc_addr = raw_addr.clone().encrypt(
+                &shared_val_netaddr_key,
```

Comment:
Nit: I don't think there's a lot of value declaring a constant a variable, and then immediately using the variable.